### ISSUE TITLE
`SkyModel.get_MIGHTEE_Sky()` gives no warning and is double-checked on correctness

### ISSUE THREAD

**Issue by kenfus** [i4Ds/Karabo-Pipeline#379]
While checking for the distribution of frequencies, I looked at GLEAM and MIGHTEE. They both have the `76e6` frequency hard-coded: 

```
survey = GLEAMSurveyDownloadObject()
path = survey.get()
gleam = SkyModel.get_fits_catalog(path)
df_gleam = gleam.to_pandas()
ref_freq = 76e6
df_gleam = df_gleam[~df_gleam["Fp076"].isna()]
ra, dec, fp = df_gleam["RAJ2000"], df_gleam["DEJ2000"], df_gleam["Fp076"]
sky_array = np.column_stack(
    (
        ra,
        dec,
        fp,
        np.zeros(ra.shape[0]),
        np.zeros(ra.shape[0]),
        np.zeros(ra.shape[0]),
        [ref_freq] * ra.shape[0],
    )
).astype("float64")
```

Same in mightee:

```
survey = MIGHTEESurveyDownloadObject()
path = survey.get()
mightee = SkyModel.get_fits_catalog(path)
df_mightee = mightee.to_pandas()
ref_freq = 76e6
ra, dec, fp = df_mightee["RA"], df_mightee["DEC"], df_mightee["NU_EFF"]
sky_array = np.column_stack(
    (
        ra,
        dec,
        fp,
        np.zeros(ra.shape[0]),
        np.zeros(ra.shape[0]),
        np.zeros(ra.shape[0]),
        [ref_freq] * ra.shape[0],
    )
).astype("float64")
sky = SkyModel(sky_array)
```

**Comment by kenfus** [comment]
Gleam was fixed in #381

**Comment by Lukas113** [comment]
@kenfus close this issue?
